dashboard/views.py

```diff
@@ -240,24 +239,23 @@ def calculate_vibe(request):
             if lyric_vibe:
                 vibe_result += " " + lyric_vibe
 
-            # Find row in vibe database that is within 24 hours
```

Comment:
So it means with HF endpoint, we no longer lookup database for recent vibe?

Reply:
We still do?

Reply:
Got it